docs/publishing.md

```diff
@@ -23,35 +24,43 @@ When a release is immenent:
 
-   If needed, you can repeat this step on the same branch, using a `minor` or `major` argument (instead of `patch`), to increment the version for newly-introduced features or breaking changes (see [_Semantic versioning policy_](../README.md#semantic-versioning-policy)).
+3. Increment the package version with `npm version --no-git-tag-version [major | minor | patch]` and commit the change.
 
-3. Run `npm run release-notes` — this command will take several minutes to finish — and copy standard output to `RELEASE_NOTES.md`.
+   For example, to increment the version for a routine data update with no breaking changes or new features, run `npm version --no-git-tag-version patch`, then commit the changes to the package metadata files.
 
-4. Run `npm run release-pulls` to get a link to all of the labeled pull requests since the previous release. Confirm that all pull requests labeled `needs-release-note 📰` have been entered into the release notes (written manually, if necessary). Remove the labels when release notes are complete.
+4. Run `npm run release-notes` — this command will take several minutes to finish — and copy standard output to `RELEASE_NOTES.md`.
 
-5. Add the release statistics to the release notes. Switch to the `main` branch, then run `npm run release-stats`. After completing the prompts, copy the output, switch back to the release branch, and add the statistics to `RELEASE_NOTES.md`.
+5. From the link provided in step 2, confirm that all pull requests labeled `needs-release-note 📰` have been entered into the release notes (written manually, if necessary).
 
-6. Confirm all `TODO` comments in the `RELEASE_NOTES.md` file are completed and removed.
+6. Add the release statistics to the release notes. Run `npm run release-stats` and complete the prompts, then copy the output and add it to `RELEASE_NOTES.md`.
 
-7. Commit your changes, push the release branch to your remote, and open a pull request on GitHub.
+7. Confirm all `TODO` comments in the `RELEASE_NOTES.md` file are completed and removed.
 
-8. Mark the pull request as ready for review.
+8. Commit your changes, push the release branch to your remote, and open a pull request on GitHub.
 
 9. Confirm that CI passes before continuing.
 
-10. If applicable or desired, seek a review, then merge the pull request.
+10. If applicable or desired, seek a review.
+
+You are now ready to perform the actual release!
+
+## Perform the release
+
+Now that you have created the PR and gotten the applicable review, you may now perform the release:
+
+1. Merge the pull request you previously created.
 
-11. Start a [release on GitHub](https://github.com/mdn/browser-compat-data/releases).
+2. Start a [release on GitHub](https://github.com/mdn/browser-compat-data/releases).
```

Comment:
A few of the steps here could be automated by a workflow triggered by the change of package.json version. Either all of it, or just creating a draft release for human review.

Reply:
I don't doubt this could be automated further, and that's the eventual plan!  For now though, these steps are still required.